### PR TITLE
87 missing shname und shproperty in properties with shnode

### PR DESCRIPTION
**Pull request by MircoNierenz**
# Description

add name and description to groups for all shacls

## Type of change

Please delete options that are not relevant.

- [ ] New type (non-breaking change which adds a type)
- [x] Change (non-breaking change or fix on an existing type)
- [ ] Breaking change (Change that would cause existing Self Descriptions not to be accepted anymore by a Federated Catalogue)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any
relevant details for your test configuration

- Test A
- Test B

## Checklist

- [x] My code follows the modelling guidelines of this project
- [x] I have performed a self-review of my own changes

### CODE REVIEW

**Review by lenasauermann — APPROVED** [review]

**Review by robertschubert — CHANGES_REQUESTED** [review]
See review remarks.
After this we need a merge from master or rebase to resolve the conflicts

File: surface-model/surface-model_shacl.ttl [hunk 1]
```diff
@@ -9,31 +9,43 @@ surface-model:SurfaceModelShape a sh:NodeShape ;
...
             sh:minCount 1 ;
             sh:node surface-model:FormatShape ;
+            sh:name "formatObject" ;
+            sh:description "format object with properties for format informations" ;
             sh:order 2 ;
             sh:path surface-model:Format ],
         [ sh:maxCount 1 ;
             sh:minCount 1 ;
             sh:node surface-model:DataShape ;
+            sh:name "DataObject" ;
```
[the remark this answers is not in the document]
Comment (MircoNierenz) [reply surface-model/surface-model_shacl.ttl]: fixed in last commit

File: environment-model/environment-model_shacl.ttl [hunk 2]
```diff
@@ -9,31 +9,43 @@ environment-model:EnvironmentModelShape a sh:NodeShape ;
     sh:property [ sh:maxCount 1 ;
             sh:minCount 1 ;
             sh:node general:GeneralShape ;
+            sh:name "generalObject" ;
```
[the remark this answers is not in the document]
Comment (MircoNierenz) [reply environment-model/environment-model_shacl.ttl]: change in last commit to "[name] object"

**Review by lenasauermann — APPROVED** [review]

**Comment by robertschubert** [comment]
Pipeline is failing, I guess due to missing merge from master. 
I will resolve the conflicts.

**Pull request merged**